filebeat/prospector/prospector_log.go

```diff
@@ -298,6 +298,8 @@ func (p *ProspectorLog) handleIgnoreOlder(lastState, newState file.State) error
 		return nil
 	}
 
+	newState.Offset = newState.Fileinfo.Size()
```

Comment:
Add a comment explaining why we set offset to filesize + why is ignored file added to registry. See PR description.